views/js/controller/creator/helpers/category.js

```diff
@@ -41,13 +39,21 @@ define([
      * @param {Function} cb
      */
     function eachCategories(testModel, cb) {
+        const getCategoriesRecursively = section => {
+            _.forEach(section.sectionParts, function (sectionPart) {
+                if (sectionPart['qti-type'] === 'assessmentItemRef') {
+                    _.forEach(sectionPart.categories, function (category) {
+                        cb(category, sectionPart);
+                    });
+                }
```

Comment:
quick question, is there a way in lodash to implement a "continue" instruction in the loop so you avoid evaluating the next if in the case of sectionPart['qti-type'] === 'assessmentItemRef'? :)